tools/README.md

```diff
@@ -362,4 +362,21 @@ For example, in order to translate a windows-related Sigma rule, one would use:
 
 ```bash
 tools/sigmac -t devo -c tools/config/devo-windows.yml rules/windows/sysmon/sysmon_wmi_susp_scripting.yml
-```
\ No newline at end of file
+```
+
+### Datadog
+The Datadog backend currently supports converting Sigma files to the [Log Search Syntax](https://docs.datadoghq.com/logs/explorer/search_syntax/)
+with the identifier `datadog-logs`. This query can be used in the Security Monitoring product.
+
+#### Config file
+This backend does not require a config file though it is possible to add a config file for fieldmapping and defining facets as a list.
```

Comment:
This mentions `fieldmapping` can be done in the backend config file, does the example config file show that functionality?

Reply:
I also think it could be added in the example config but no fieldmapping is required by default (not sure of that). Maybe fieldmapping can be added in the example config with comments ?